docs/auditing.md

```diff
@@ -2,6 +2,8 @@
 
 The component guide includes some auditing of components and applications that can provide useful information. It can be found at `/component-guide/audit`.
 
+It is designed to be run locally as it gets information about applications and how they use components from local files. Note that if your local copy of an application is not up to date the information reported by the auditing may be incorrect.
```

Comment:
I think that the local apps need to be up to date is worth highlighting in a way that can't be missed - not necessarily as I've suggested, just in any way that highlights it strongly:

```suggestion
It is designed to be run locally as it gets information about applications and how they use components from local files. 

> **If your local copy of an application is not up to date the information reported by the auditing may be incorrect**
```

Reply:
Thanks, have highlighted this (I couldn't think of a better way than you'd suggested, it's clear at least).